### ISSUE TITLE
Translation hot keys have no effect

### ISSUE THREAD

**Issue by rosskevin** [BuzzteeBear/OpenXR-MotionCompensation#6]
I tried the default translation hot keys to no effect.  I changed their mapping to something very unique twice, still no change.  Rotational hot keys worked alongside, but could never get translation to work.  I heard audible feedback on rotation but not translation.

v0.2.7

**Comment by BuzzteeBear** [comment]
Can't reproduce this on my end. Default key combination seems to work fine. Which application do have this issue with?

**Comment by rosskevin** [comment]
I'm using xpadder as an intermediary to trigger the keys.  Every other combination of keys worked, so it is strange that only translation did not.  I did see the visual trigger via xpadder from the dial activation, so I know that connection is good, and I remapped/saved/reloaded config (three different sets of key combinations) before returning them to the default.

Should I receive an audible confirmation like the rotation?
